app/uk/gov/hmrc/customs/financials/emailthrottler/services/EmailNotificationService.scala

```diff
@@ -36,18 +38,22 @@ class EmailNotificationService @Inject()(http: HttpClient, metricsReporter: Metr
     implicit val hc: HeaderCarrier = HeaderCarrier()
 
     metricsReporter.withResponseTimeLogging("email.post.send-email") {
-      http.POST[EmailRequest, HttpResponse](appConfig.sendEmailUrl, request).collect {
-        case response if response.status == Status.ACCEPTED =>
-          log.info(s"[SendEmail] Successful for ${request.to}")
-          true
-        case response =>
-          log.error(s"[SendEmail] Failed for ${request.to} with status - ${response.status} error - ${response.body}")
+      http.post(url"${appConfig.sendEmailUrl}")
+        .withBody[EmailRequest](request)
+        .execute[HttpResponse]
+        .flatMap {
+          case response if response.status == Status.ACCEPTED => Future.successful(log.info(
+            s"[SendEmail] Successful for ${request.to}"))
+          Future.successful(true)
```

Comment:
Add a line break after this line to differentiate from other case and to make it readable. Same for line 51